File: Android/termwork 3/app/src/main/java/com/example/simpletito/MainActivity.java
package com.example.simpletito;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Toast;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.google.android.material.floatingactionbutton.FloatingActionButton;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {
    private List<videoItem> myVideoList=new ArrayList<>();//TODO:fetch the data from api

    private FloatingActionButton takeVideoBtn;

    @Override
    protected void onCreate(@Nullable @org.jetbrains.annotations.Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.main_activity_layout);

        takeVideoBtn = findViewById(R.id.btn_take_video);
        takeVideoBtn.setOnClickListener(v -> {
            Toast.makeText(MainActivity.this, "拍摄视频", Toast.LENGTH_SHORT).show();
            Intent intent = new Intent(MainActivity.this, TakeVideoActivity.class);
            startActivity(intent);
        });
    }

    private void initVideoList(){

    }//TODO:Add your code here(maybe)
}

File: Android/AndroidArch/AndroidArch/mvvm-with-jetpack-app/src/main/java/com/android/example/androidarch/MainActivity.kt
package com.android.example.androidarch

import android.os.Bundle
import android.widget.Button
import androidx.activity.viewModels
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    // To use the viewModels() extension function, include
    // "androidx.fragment:fragment-ktx:latest-version" in your app
    // module's build.gradle file.
    private val viewModel: CountViewModel by viewModels()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val button = findViewById<Button>(R.id.btn_click)
        button.setOnClickListener {
            viewModel.count.value = viewModel.count.value!! + 1
        }
        viewModel.count.observe(this) {
            button.text = "Click $it times!"
        }
    }
}